score-lang-cli/src/main/java/org/openscore/lang/cli/services/SyncTriggerEventListener.java

```diff
@@ -93,6 +93,7 @@ public synchronized void onEvent(ScoreEvent scoreEvent) throws InterruptedExcept
                 break;
 
             case EVENT_EXECUTION_FINISHED :
+                flowFinished.set(true);
```

Comment:
On the contrary, I think we should stop listening to score finished event. And only to slang event.